README.md

```diff
@@ -265,6 +267,29 @@ ln -s ha-google-home/custom_components/google_home ~/.homeassistant/custom_compo
 
 Make sure that you have your Home Assistant Container network set to `host`, as perscribed in the official docker installation for Home Assistant.
 
+### ARM Docker Container Dependencies Workaround
+
+If you are installing this integration on an ARM based device (Like Raspberry Pi, Synology, etc), you may need to do the following if you get this error:
+
+```
+ERROR: Cannot install glocaltokens==0.3.1
+```
+Please run the following command in the Home Assistant container to add the missing dependencies for `glocaltokens`:
+
+```
+apk add gcc g++
+```
+then you can install `glocaltokens` manually like this:
+```
+pip3 install glocaltokens
+```
+
+This will need to be done each time the image is updated unfortunately. Alternatively you can add the command to the container startup:
```

Comment:
```suggestion
Unfortunately, this will need to be done each time the image is updated. Alternatively you can add the command to the container startup:
```